data/data-pipeline/data_pipeline/etl/datasource.py

```diff
@@ -0,0 +1,112 @@
+from pathlib import Path
+from typing import Protocol, List
+from dataclasses import dataclass
+from abc import ABC, abstractmethod
+
+from data_pipeline.etl.downloader import Downloader
+from data_pipeline.etl.sources.census_acs.etl_utils import (
+    retrieve_census_acs_data,
+)
+
+
+class Fetchable(Protocol):
```

Comment:
I love protocols, but this isn't used anywhere and I think the ABC would fulfill the same purposes for typechecking?

Reply:
I think you're right. I had bigger plans but scrapped them for simplicity. This is a vestigial tail that can be removed.